packages/desktop-client/src/components/settings/Themes.tsx

```diff
@@ -91,7 +92,8 @@ export function ThemeSettings() {
       }
     >
       <Text>
-        <strong>Themes</strong> change the user interface colors.
+        <strong>{t('Themes')} </strong>
+        {t('change the user interface colors.')}
```

Comment:
I think this is the last of it: 

```suggestion
        {t(' change the user interface colors.')}
```